python/scannerpy/table.py

```diff
@@ -37,13 +37,14 @@ def columns(self, index=None):
         return columns[index] if index is not None else columns
 
     def num_rows(self):
-        return self._descriptor.num_rows
+        return self._descriptor.end_rows[-1]
```

Comment:
Why would the number of rows in the table be the same as the last row? If my table had rows [0, 100, 200], then it doesn't have 200 rows.